app/controllers/bets_controller.rb

```diff
@@ -1,5 +1,22 @@
 class BetsController < ApplicationController
-  def new_bet
+  def new
     @bet = Bet.new
   end
+
+  def create
+    @bet = Bet.new(bet_params)
+    if @bet.save
+      flash[:success] = 'Bet has been created'
+      redirect_to root_path
```

Comment:
Should redirect to the Bet#show page
Same with update when you get there. 

After you create something, as a user, dont you want to see the thing you created?